util/kube/ctl_test.go

```diff
@@ -10,6 +11,24 @@ import (
 )
 
 func TestConvertToVersion(t *testing.T) {
+	/*
```

Comment:
@dthomson25 this test doesn't do anything with Argo CD? It just tests `kubectl`?